src/ansys/mapdl/core/mapdl.py

```diff
@@ -491,13 +491,28 @@ def queries(self):
     def non_interactive(self):
         """Non-interactive context manager.
 
-        Use this when using commands that require user
-        interaction within MAPDL (e.g. :func:`Mapdl.vwrite`).
+        Allows to execute code without user interaction or waiting
```

Comment:
```suggestion
        Allow to execute code without user interaction or waiting
```

Reply:
From what I saw, the documentation seems to use infinitives.

Reply:
Thank you a lot Camille! I am really bad at this! 🥲